src/components/common/seo.tsx

```diff
@@ -1,24 +1,38 @@
 import React from "react";
-import PropTypes from "prop-types";
 import { Helmet } from "react-helmet";
 import { useStaticQuery, graphql } from "gatsby";
 
-function SEO({ description, lang, meta, title }) {
-  const { site } = useStaticQuery(
+const getSocialImageSrc = (image: { src; alt }, site) => {
```

Comment:
Would be good to strengthen the typing around the signatures of the functions touched by this PR where possible, both input parameters and return types.

Reply:
Done